GooglePay/src/main/java/com/braintreepayments/api/GooglePayTokenizationParameters.kt

```diff
@@ -0,0 +1,20 @@
+package com.braintreepayments.api
+
+import com.google.android.gms.wallet.PaymentMethodTokenizationParameters
+
+/**
+ * A request used to launch the Venmo app for continuation of the Venmo payment flow.
+ */
+sealed class GooglePayTokenizationParameters {
+
+    /**
+     * The request was successfully created and is ready to be launched by [VenmoLauncher]
+     */
+    class Success(val parameters: PaymentMethodTokenizationParameters, val allowedCardNetworks:
```

Comment:
This class might read a little easier with the vals on separate lines:

```kotlin
class Success(
    val parameters: PaymentMethodTokenizationParameters,
    val allowedCardNetworks: Collection<Integer>
) : GooglePayTokenizationParameters()
```

Reply:
Fixed 👍